x-pack/metricbeat/module/googlecloud/compute/_meta/data.json

```diff
@@ -0,0 +1,77 @@
+{
+    "_index": "metricbeat-8.0.0-2019.11.27-000001",
+    "_id": "H4iRrm4B7vvv-udOwwFO",
+    "_version": 1,
+    "_score": null,
+    "_source": {
+        "@timestamp": "2019-11-27T20:30:00.000Z",
+        "ecs": {
+            "version": "1.2.0"
+        },
+        "host": {
+            "name": "mcastro"
+        },
+        "agent": {
+            "version": "8.0.0",
+            "type": "metricbeat",
+            "ephemeral_id": "cb56f25c-fc0e-4bbc-9c49-4551c1a97ab5",
+            "hostname": "mcastro",
+            "id": "7e36a073-1a32-4a94-b65b-4c7f971fb228"
+        },
+        "cloud": {
+            "availability_zone": "us-central1-a",
+            "account": {
+                "id": "elastic-metricbeat"
+            },
+            "provider": "googlecloud",
+            "instance": {
+                "id": "3296438752514919533",
+                "name": "instance-3"
+            }
+        },
+        "event": {
+            "module": "googlecloud",
+            "duration": 1252249339,
+            "dataset": "googlecloud.compute"
+        },
+        "metricset": {
+            "period": 300000,
+            "name": "compute"
+        },
+        "service": {
+            "type": "googlecloud"
+        },
+        "googlecloud": {
+            "compute": {
+                "labels": {
+                    "user": {
+                        "label_key": "label_value"
+                    },
+                    "metadata": {
+                        "metadata_key": "metadata_value"
+                    }
+                },
+                "instance": {
+                    "uptime": 60,
+                    "cpu": {
+                        "usage_time": 0.049102112533603304,
```

Comment:
add the unit after `instance.cpu.usage_time` will help. Maybe `instance.cpu.usage_time.ms`?

Reply:
Good suggestion. I can't agree more. Unfortunately, because we are using lightweight modules; this is a limitation we have to deal with (the metric name is the same than in google cloud). I was considering to setup a `map` of name conversions but I thought that it might be a bit too much

Reply:
Short-term we could consider implementing mapping on each generic module, but mid-long term, better have something that can be reused in all modules, like https://github.com/elastic/beats/issues/14740